package/Dockerfile.dapper-base

```diff
@@ -40,4 +42,6 @@ RUN dnf -y distrosync && \
     tar -xzf kubefedctl-${KUBEFED_VERSION}-linux-${ARCH}.tgz && cp kubefedctl /usr/bin/ && chmod a+x /usr/bin/kubefedctl && \
     curl -Lo /usr/bin/kind "https://github.com/kubernetes-sigs/kind/releases/download/${KIND_VERSION}/kind-linux-${ARCH}" && chmod a+x /usr/bin/kind && \
     GOFLAGS="" go get -v github.com/onsi/ginkgo/ginkgo && \
-    GOFLAGS="" go get -v golang.org/x/tools/cmd/goimports
+    GOFLAGS="" go get -v golang.org/x/tools/cmd/goimports && \
+    upx /usr/bin/kubectl /usr/bin/dockerd /usr/bin/docker /usr/bin/containerd /usr/bin/helm /usr/bin/kubefedctl && \
```

Comment:
You can also shrink the following:
/go/bin/golangci-lint
/usr/bin/ctr
/linux-amd64/tiller
/usr/bin/runc

Also for some reason these seem to be duplicate:
40840	/linux-amd64/helm
40840	/usr/bin/helm

Reply:
Thanks, I got rid of the dupe and compressed the other binaries.